tests/core/gadget-config-defaults/task.yaml

```diff
@@ -117,14 +117,11 @@ restore: |
     restore_pc_snap
 
     TEST_REVNO=$(awk "/^snap-revision: / {print \$2}" "test-snapd-with-configure${SUFFIX}"_*.assert)
-    if systemctl status "$(systemd-escape --path "/snap/test-snapd-with-configure${SUFFIX}/$TEST_REVNO.mount")"; then
```

Comment:
It is needed to update the test gadget-config-defaults-vitality as well.

Reply:
Done.

Reply:
tx